.github/workflows/tests.yml

```diff
@@ -9,11 +9,96 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/create-build
+        with:
+          target: ${{ matrix.target }}
+      - uses: actions/upload-artifact@v2
```

Comment:
This is only needed to share artifacts with the `install_head` job right? 

If so, it might just make sense to fuse these two jobs together.

Reply:
It's also used for the `vscode-test` job but if we remove that it would make sense to fuse them.

Reply:
The artiface is used in the `test-vscode` job but if we remove that we can fuse these two together.